cs-dropbox/src/main/java/io/cloudslang/content/dropbox/utils/InputsUtil.java

```diff
@@ -0,0 +1,132 @@
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.dropbox.utils;
+
+import io.cloudslang.content.dropbox.entities.inputs.InputsWrapper;
+import io.cloudslang.content.httpclient.HttpClientInputs;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static io.cloudslang.content.constants.BooleanValues.FALSE;
+import static io.cloudslang.content.dropbox.entities.constants.Constants.ErrorMessages.CONSTRAINS_ERROR_MESSAGE;
+import static io.cloudslang.content.dropbox.entities.constants.Constants.HttpClientInputsValues.ALLOW_ALL;
+import static io.cloudslang.content.dropbox.entities.constants.Constants.HttpClientInputsValues.BROWSER_COMPATIBLE;
+import static io.cloudslang.content.dropbox.entities.constants.Constants.HttpClientInputsValues.STRICT;
+import static io.cloudslang.content.dropbox.entities.constants.Constants.Miscellaneous.PATH_REGEX;
+import static io.cloudslang.content.dropbox.entities.constants.Constants.Values.INIT_INDEX;
+import static io.cloudslang.content.dropbox.factory.UriFactory.getUri;
+import static io.cloudslang.content.httpclient.build.auth.AuthTypes.ANONYMOUS;
+import static io.cloudslang.content.utils.BooleanUtilities.isValid;
+import static java.lang.Boolean.parseBoolean;
+import static java.lang.String.format;
+import static java.lang.String.valueOf;
+import static java.util.Arrays.asList;
+import static java.util.regex.Pattern.compile;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+/**
+ * Created by TusaM
+ * 5/26/2017.
+ */
+public class InputsUtil {
+    private InputsUtil() {
+        // prevent instantiation
+    }
+
+    public static HttpClientInputs getHttpClientInputs(String proxyHost, String proxyPort, String proxyUsername,
+                                                       String proxyPassword, String trustAllRoots, String x509HostnameVerifier,
+                                                       String trustKeystore, String trustPassword, String keystore,
+                                                       String keystorePassword, String connectTimeout, String socketTimeout,
+                                                       String useCookies, String keepAlive, String method) {
+        HttpClientInputs httpClientInputs = new HttpClientInputs();
+        httpClientInputs.setAuthType(ANONYMOUS);
+        httpClientInputs.setMethod(method);
+        httpClientInputs.setQueryParamsAreURLEncoded(FALSE);
+
+        httpClientInputs.setTrustAllRoots(valueOf(getEnforcedBooleanCondition(trustAllRoots, true)));
```

Comment:
@Tirla-Alin Because in open-source I think the base security level (default values) should be as light as possible/permissive/out of the box ... If I want to secure my app/communications then I would prepare an appropriate setup for (including trust all roots, host name verifier, certificates and so on...). But that's just me. If (for some reason) should be otherwise please feel free to modify...